### PR TITLE
fixed swapPanel height

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: src/app/components/side-panel.js [hunk 1]
```diff
@@ -29,8 +31,9 @@ const css = csjs`
     cursor: pointer;
   }
   .pluginsContainer {
-    height: calc(100% - 35px);
-    overflow: auto;
+    height: 100%;
```
Comment (GrandSchtroumpf) [inline src/app/components/side-panel.js]: remove height here. flex: 1 should be enough.

File: src/app/components/side-panel.js [hunk 2]
```diff
@@ -29,8 +31,9 @@ const css = csjs`
...
+    flex: 1;
+    padding-bottom: 40px;
```
Comment (GrandSchtroumpf) [inline src/app/components/side-panel.js]: Why ?

File: src/app/components/side-panel.js [hunk 3]
```diff
@@ -5,7 +5,9 @@ const yo = require('yo-yo')
 const css = csjs`
   .panel {
     height: 100%;
-    overflow-y: hidden; 
+    overflow-y: hidden;
```
Comment (GrandSchtroumpf) [inline src/app/components/side-panel.js]: remove this. There is no reason to hide it now.

File: src/app/components/side-panel.js (hunk 1 above)
Comment (LianaHus) [reply src/app/components/side-panel.js]: no it's not

File: src/app/components/side-panel.js (hunk 2 above)
Comment (LianaHus) [reply src/app/components/side-panel.js]: because otherwise the top is shifted when it's not an iframe

**Pull request merged**